app/helpers/compliance_summary_helper.rb

```diff
@@ -25,4 +29,23 @@ def textual_compliance_status
     end
     h
   end
+
+  def textual_compliance_history(options_if_available = {})
+    h = {:label => _("History")}
+    if @record.number_of(:compliances) == 0
+      h[:value] = _("Not Available")
+    else
+      h[:image] = "compliance"
+      h[:value] = _("Available")
+      h[:title] = _("Show Compliance History of this %{model} (Last 10 Checks)") %
+                  {:model => ui_lookup(:model => controller.class.model.name)}
+      h[:link] = url_for(
+        :controller => controller.controller_name,
+        :action     => 'show',
+        :id         => @record,
+        :display    => 'compliance_history')
+      h.merge!(options_if_available)
```

Comment:
I would merge the options when returning h so we merge when number of compliances  is 0 too.